_clients/python-low-level.md

```diff
@@ -98,6 +98,41 @@ client = OpenSearch(
 ```
 {% include copy.html %}
 
+If you need to verify ssl to a different hostname, specify it in `ssl_assert_hostname`: 
+
+```python
+host = 'localhost'
+port = 9200
+
+# Create the client with SSL/TLS and hostname verification disabled.
+client = OpenSearch(
+    hosts = [{'host': host, 'port': port}],
+    http_compress = True, # enables gzip compression for request bodies
+    use_ssl = True,
+    verify_certs = False,
+    ssl_assert_hostname = 'example.com',
+    ssl_show_warn = False
+)
+```
+{% include copy.html %}
+
+If you need to enable ssl hostname verification to the same host, specify None, or leave it to its default value.
```

Comment:
I believe this case isn't necessary. We can just add it as a comment in the previous case, as seen in this [PR](https://github.com/opensearch-project/opensearch-py/pull/694/files#diff-c30be5e16358678f3c4a1a369fb517f6869a129560d7b04fb9144b64813a1d01R57). Feel free to share your thoughts on the PR if you have any other ideas. Thank you :)